compiler/resolution/functionResolution.cpp

```diff
@@ -7936,6 +7928,31 @@ static void handleRuntimeTypes()
   insertRuntimeInitTemps();
 }
 
+//
+// If a symbol's type is not live, we can't use it.
+// Remove it if VarSymbol; remove containing fn if ArgSymbol.
+// Hopefully they are not used.
+//
+static void removeSymbolsWithRemovedTypes() {
+  std::vector<Symbol*> removedSyms; // for verification
```

Comment:
I think it'd make more sense to include the 1-line ASSERT below just before the remove call, and not have a vector at all